shell/platform/tizen/channels/app_control_channel.cc

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include "app_control_channel.h"
+#include "encodable_value_getter.h"
```

Comment:
For consistency,
```suggestion
#include "flutter/shell/platform/tizen/channels/encodable_value_getter.h"
```
(you need to sort the includes properly after making the change)

Reply:
Thanks!